### PR TITLE
Feature/kids 47 add correlation id to headers

### PR DESCRIPTION
**Pull request by Daniela510**
## Description

- Create a guid in the http interceptor and add it to the header of the intercepted call

### CODE REVIEW

**Review by pavelpavelko — APPROVED** [review]

File: lib/core/app/app_router.dart [hunk 1]
```diff
@@ -129,8 +129,8 @@ class AppRouter {
           },
         ),
         GoRoute(
-          path: Pages.organisations.path,
-          name: Pages.organisations.name,
+          path: Pages.recommendedOrganisations.path,
```
Comment (MaikelStuivenberg) [inline lib/core/app/app_router.dart:132]: Was this something wrong? Or why is this changed? :)
